packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -208,7 +208,10 @@ function InserterMenu( {
 		>
 			<BottomSheetConsumer>
 				{ ( { listProps } ) => (
-					<TouchableHighlight accessible={ false }>
+					<TouchableHighlight
+						accessible={ false }
+						style={ styles[ 'inserter-tabs__wrapper' ] }
```

Comment:
minor nitpick: The `TouchableHighlight` wraps both the `InserterTabs` and the `InserterSearchResults` so I'm thinking we might need a different _block_ name to capture both components, maybe `inserter-menu__content-wrapper` ?

Reply:
Ah, good point. I did not notice the confusing naming context. I agree that name oriented to `inserter-menu` makes sense. 

I'd propose we describe the type of content further. E.g. `inserter-menu__list-wrapper`. WDYT?

Reply:
> I'd propose we describe the type of content further. E.g. inserter-menu__list-wrapper. WDYT?
Perfect!